src/core/lv_refr.c

```diff
@@ -291,6 +291,12 @@ void lv_inv_area(lv_display_t * disp, const lv_area_t * area_p)
     suc = lv_area_intersect(&com_area, area_p, &scr_area);
     if(suc == false)  return; /*Out of the screen*/
 
+    if(disp->color_format == LV_COLOR_FORMAT_I1) {
+        com_area.x1 &= ~0x7; /*Round down*/
+        com_area.x2 += 7;    /*Round up*/
+        com_area.x2 &= ~0x7;
+    }
```

Comment:
True, the rounder callback was missing.

Reply:
Is x2 "the first unused position" (comparison `foo<x2`) or "the last used position" (comparison `foo<=x2`)? In the latter case, lines 296 and 297 must be exchanged for correct rounding.

Reply:
Should be fixed now